### PR TITLE
Ishita1608 euler's totient function in java

### PR DESCRIPTION
**Pull request by ishitakeshawani**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

I add Solution of Euler's totient function in java

### Checklist

- [✔ ] I've read the contribution guidelines.
- [✔ ] I've checked the issue list before deciding what to submit.
- [✔ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This is the solution of the issue no #445

### CODE REVIEW

**Review by siddharth25pandey — APPROVED** [review]
@Ishita1608 looks fine to me
Able to merge.
Thanks!

File: Java/cp/Euler.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// Java program to calculate 
+// Euler's Totient Function 
+import java.io.*; 
+import java.util.*;
+
+class Euler 
+{ 
+static int phi(int n)
```
Comment (vatsalkesarwani12) [inline Java/cp/Euler.java:8]: make a practice of naming the function with some logical name

File: Java/cp/Euler.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	// greater than sqrt(n) 
+	// (There can be at-most 
+	// one such prime factor) 
+	if (n > 1) 
+		result -= result / n; 
+	return result; 
+} 
+
+
+public static void main (String[] args)
```
Comment (vatsalkesarwani12) [inline Java/cp/Euler.java:42]: correct the indentation

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
https://www.geeksforgeeks.org/eulers-totient-function/

Code copied

**Pull request closed without merging**